web/templates/verify.html

```diff
@@ -9,7 +9,7 @@
         </div>
     </div>
     <script>
-        window.setup_verify_page('{{ .Props.IsVerified }}');
+window.setupVerifyPage('{{.Props.IsVerified}}', '{{.Props.TeamURL}}', '{{.Props.UserEmail}}');
```

Comment:
My vim hates html files. It's always trying to do that to any JS in the html files.